med-web-app-front/src/services/dicom-anonymizer.service.js

```diff
@@ -32,8 +35,7 @@ class DicomAnonymizerService {
         this.anonymizeTag(dicomTagsDataset, 'StudyDate', '00/00/0000');
 
         dicomContent.dict = dcmjs.data.DicomMetaDictionary.denaturalizeDataset(dicomTagsDataset);
-
-        return dicomContent.write();
+        return {dicom: dicomContent.write(), UID: dicomTagsDataset.StudyInstanceUID};

```

Comment:
Codacy has a fix for the issue: [Replace `········return·{dicom:·dicomContent.write(),·UID:·dicomTagsDataset.StudyInstanceUID};␍` with `····return·{⏎······dicom:·dicomContent.write(),⏎······UID:·dicomTagsDataset.StudyInstanceUID,`](https://app.codacy.com/gh/danielsheh02/Medical-Web-App/pullRequest?prid=9534848)
```suggestion
    return {
```